acquia_lift.install

```diff
@@ -639,6 +638,186 @@ function acquia_lift_update_7017() {
 }
 
 /**
+ * Miscellaneous changes required for the new workflow, includes converting legacy
+ * campaigns to "personalizations".
+ */
+function acquia_lift_update_7018() {
```

Comment:
You and Shay have the same update number in your update hooks.... something to watch out for in merging.